std/string.d

```diff
@@ -2267,7 +2239,7 @@ unittest
         assert(s2 !is s1);
         s1 = to!S("\u0131 \u0130");
         s2 = capitalize(s1);
-        assert(cmp(s2, "\u0049 \u0069") == 0);
+        assert(cmp(s2, "\u0049 i\u0307") == 0);
```

Comment:
Be careful here. There *are* some places in Unicode where the uppercase of the lowercase of something is not equal to the original thing.  I vaguely recall Ali Cehreli mentioning before certain issues involving the upper/lower-casing of the Turkish dotted/undotted i's. Are you absolutely certain this isn't an instance of that?